src/mkdocs_jupyter/plugin.py

```diff
@@ -1,9 +1,10 @@
 import os
 import pathlib
 
+import jupytext
 import markdown
-from markdown.extensions.toc import TocExtension
 import mkdocs
+from markdown.extensions.toc import TocExtension
```

Comment:
i didn't do these btw... must have been from `just fmt`... are those not run during CI?

Reply:
oh weird. Maybe from your editor? No actions have run yet on this PR and the actions only report if something is broken they dont modify the code.

Reply:
No no, sorry I _did_ run just fmt locally, I was just surprised that it was changing stuff that I didn't touch :)

Reply:
Ah ok, then yes. it came from that then :)